src/Server/Query.elm

```diff
@@ -257,6 +257,35 @@ validateMassInGrams string =
         |> Result.map Mass.grams
 
 
+validatePhysicalDurability : String -> Result String Unit.PhysicalDurability
+validatePhysicalDurability string =
+    string
+        |> String.toFloat
+        |> Result.fromMaybe ("Durabilité invalide\u{202F}: " ++ string)
+        |> Result.andThen
+            (\durability ->
+                let
+                    minFloatDurability =
+                        Unit.minDurability Unit.PhysicalDurability |> Unit.physicalDurabilityToFloat
+
+                    maxFloatDurability =
+                        Unit.maxDurability Unit.PhysicalDurability |> Unit.physicalDurabilityToFloat
+                in
+                if durability < minFloatDurability || durability > maxFloatDurability then
+                    Err
+                        ("La durabilité doit être comprise entre "
+                            ++ String.fromFloat minFloatDurability
+                            ++ " et "
+                            ++ String.fromFloat maxFloatDurability
+                            ++ "."
+                        )
```

Comment:
For some reason this doesn't seem to work: 
![image](https://github.com/user-attachments/assets/47c1b414-ced5-484e-bd7a-83da03b60c18)

Also, we'll probably want a unit test in `ServerTest.elm` to check that validation works

Reply:
It also appears that the new field is not documented for the POST endpoint in the OpenAPI schema 
![image](https://github.com/user-attachments/assets/6c6b743e-b290-46c7-83c5-a231e6d424f8)

Reply:
I've added the param docs for POST in da0fff1b56f8230f0a47184e1fb85492756d75af

Reply:
For the lack of validation in POST, this is a MUCH broader issue I've documented in [this comment](https://github.com/MTES-MCT/ecobalyse/pull/721#issuecomment-2340120129)